build/configs/rtl8730e/loadable_ext/defconfig

```diff
@@ -552,6 +552,9 @@ CONFIG_AUDIO_ALC1019=y
 # LCD Driver Support
 #
 CONFIG_LCD=y
+CONFIG_LCD_DEVPATH="/dev/lcd0"
```

Comment:
Did you define this in os/drivers/Kconfig? If you add it here, you should also add that in the Kconfig file.
Or else, you should change it to #define LCD_DEVPATH "/dev/lcd0" in some header files.